passthrough/delivery/http/passthrough_handler.go

```diff
@@ -26,14 +29,21 @@ func formatPassthroughResource(resource string) string {
 }
 
 // NewPassthroughHandler will initialize the pools/ resources endpoint
-func NewPassthroughHandler(e *echo.Echo, ptu mvc.PassthroughUsecase, ou mvc.OrderBookUsecase) {
+func NewPassthroughHandler(e *echo.Echo, ptu mvc.PassthroughUsecase, ou mvc.OrderBookUsecase, logger log.Logger) {
 	handler := &PassthroughHandler{
 		PUsecase: ptu,
 		OUsecase: ou,
+		Logger:   logger,
 	}
 
 	e.GET(formatPassthroughResource("/portfolio-assets/:address"), handler.GetPortfolioAssetsByAddress)
 	e.GET(formatPassthroughResource("/active-orders"), handler.GetActiveOrders)
+	e.GET(formatPassthroughResource("/active-orders"), func(c echo.Context) error {
+		if c.QueryParam("sse") != "" {
+			return handler.GetActiveOrdersStream(c) // Server-Sent Events (SSE)
+		}
```

Comment:
Is there any consideration that we should make wrt load balancing?

Reply:
For a reference: discussed via slack.